data-prepper-plugins/mutate-event-processors/src/main/java/org/opensearch/dataprepper/plugins/processor/mutateevent/ConvertEntryTypeProcessorConfig.java

```diff
@@ -20,6 +20,12 @@ public class ConvertEntryTypeProcessorConfig  {
     @JsonProperty("type")
     private TargetType type = TargetType.INTEGER;
 
+    /**
+     * Optional scale value used only in the case of BigDecimal converter
+     */
+    @JsonProperty("scale")
```

Comment:
Should we expose this? What would be the value of changing the scale to something else?

You could remove the field and keep the `getScale()` method such that we don't expose the config, but are flexible to use in the future.

Reply:
As I listed in the PR description, I thought, we can let the user choose the scale he need per column like 
```processor:
   - convert_entry_type:
       key: "column1"
       type: "bigdecimal"
       scale: 5